pkg/services/rabbitmq/consumer.go

```diff
@@ -102,6 +102,11 @@ func (c *Consumer) getChannel() error {
 			log.Printf("connection to rabbitmq lost. %s", er.Error())
 			log.Printf("attempting to create new rabbitmq channel.")
 			err := c.getChannel()
+
+			if c.callback != nil {
```

Comment:
No!
 I'm not sure how I can be any clearer with this.
  `callback.consume` needs to be called if creating the channel is successful.  So this needs to move to within the below if stanza that checks if `c.getChannel()` worked (no error returned)